logger.go

```diff
@@ -127,3 +127,16 @@ func (logger *Logger) Fatal(args ...interface{}) {
 func (logger *Logger) Fatalf(format string, args ...interface{}) {
 	logger.entry().Fatalf(format, args...)
 }
+
+// SetLogLevelFromEnv allows to Configure Logger from Environment Variable with extra configuration.
+func (logger *Logger) SetLogLevelFromEnv(env string, args ...LoggerOption) error {
```

Comment:
I think the API would be neater (& more consistent) if this was an option instead

```go
// file: opts.go
func WithLogLevelFromEnv(envVarName string) LoggerOption {
 // implementation
}
```